cypress/e2e/navigation.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Navigation tests', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should navigate to search page, type a query, navigate to chain details, and go back to home on breadcrumb', () => {
+    cy.get('[data-test="search-bar"]').click();
```

Comment:
i would recommend using our `getByTestId` command
```suggestion
    cy.getByTestId('search-bar').click();
```